colossalai/nn/layer/moe/_operation.py

```diff
@@ -18,35 +15,35 @@
 class AllGather(torch.autograd.Function):
 
     @staticmethod
-    def forward(ctx: Any, inputs: Tensor, parallel_mode: ParallelMode) -> Tensor:
+    def forward(ctx: Any, inputs: Tensor, group=None) -> Tensor:

```

Comment:
group : Optional[ParallelGroup] = None?
you missed the typing hit here

Reply:
done